src/extra/AutoUI/utils.ts

```diff
@@ -56,6 +56,8 @@ export const getTagsDisabledReason = <T extends AutoUIBaseResource<T>>(
 		// TODO: Pass the resource name instead.
 		return t('info.edit_tag_no_permissions', { resource: 'item' });
 	}
+
+	return false;
```

Comment:
We let this slide without a return before? 😱

Reply:
Apparently yes :)